packages/manager/modules/octavia-load-balancer/src/onboarding/controller.js

```diff
@@ -25,4 +43,13 @@ export default class OctaviaLoadBalancerOnboardingCtrl {
       type: 'navigation',
     });
   }
+
+  createLoadBalancer($event) {
+    if (this.hasPrivateNetwork) {
+      this.$state.go('octavia-load-balancer.create');
```

Comment:
```suggestion
      this.goToLoadBalancerCreation()
```
Move $state.go in the state resolve section